Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -1072,57 +1053,68 @@ namespace GradientSignal
         LmbrCentral::DependencyNotificationBus::Event(GetEntityId(), &LmbrCentral::DependencyNotificationBus::Events::OnCompositionChanged);
     }
 
-    void ImageGradientComponent::SetPixelValueByPosition(const AZ::Vector3& position, float value)
+    PixelIndex ImageGradientComponent::GetPixelIndexForPositionInternal(const AZ::Vector3& position) const
     {
-        SetPixelValuesByPosition(AZStd::span<const AZ::Vector3>(&position, 1), AZStd::span<float>(&value, 1));
-    }
+        const auto& width = m_imageDescriptor.m_size.m_width;
+        const auto& height = m_imageDescriptor.m_size.m_height;
```

Comment:
uber nit: Do these need to be references when they types are just built-in?

Reply:
Changed.